src/components/bootcamp-dimitar/editForm.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import { useRouter } from 'next/router'
+import { FormikHelpers } from 'formik'
+import { useMutation } from 'react-query'
+import { useTranslation } from 'next-i18next'
+import { AxiosError } from 'axios'
+import makeStyles from '@mui/styles/makeStyles'
+import createStyles from '@mui/styles/createStyles'
+import { AlertStore } from 'stores/AlertStore'
+import GenericForm from 'components/common/form/GenericForm'
+import SubmitButton from 'components/common/form/SubmitButton'
+import FormTextField from 'components/common/form/FormTextField'
+import { ApiErrors, isAxiosError, matchValidator } from 'common/api-errors'
+import { editBootcampDimitar } from 'common/rest'
+import { Grid, Typography } from '@mui/material'
+
+const useStyles = makeStyles((theme) =>
+  createStyles({
+    heading: {
+      marginBottom: theme.spacing(5),
+      color: theme.palette.primary.dark,
+      textAlign: 'center',
+    },
+    message: {
+      '& textarea': { resize: 'vertical' },
+    },
+  }),
+)
+
+export default function EditForm({ data }: { data: any }) {
```

Comment:
The name of the file does not match the name of the default exported component. This may lead to pool IDE auto-suggestions and generally, it does not follow the project conventions.